tests/bluetooth/tester/src/btp_micp.c

```diff
@@ -0,0 +1,534 @@
+/* btp_micp.c - Bluetooth MICP Tester */
+
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stddef.h>
+#include <errno.h>
+
+#include <zephyr/types.h>
+#include <zephyr/kernel.h>
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/micp.h>
+#include <zephyr/bluetooth/audio/aics.h>
+
+#include "bap_endpoint.h"
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/byteorder.h>
+
+#define LOG_MODULE_NAME bttester_micp
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL
+);
+
+#include "btp/btp.h"
+
+static struct bt_micp_mic_ctlr *mic_ctlr;
+#if defined(CONFIG_BT_MICP_MIC_CTLR_AICS)
+static struct bt_micp_included micp_included;
+
+#endif /* CONFIG_BT_MICP_MIC_CTLR_AICS */
+
+static void btp_send_micp_found_ev(struct bt_conn *conn, uint16_t mute_handle)
+{
+	struct btp_micp_discovered_ev ev;
+	struct bt_conn_info info;
+
+	(void) bt_conn_get_info(conn, &info);
+	bt_addr_le_copy(&ev.address, info.le.dst);
+
+	ev.mute_handle = mute_handle;
```

Comment:
sys_cpu_to_le16()

every uint16, uint32 and uint64 needs to be converted to LE when sending over BTP